GeoEntity.md

```diff
@@ -0,0 +1,280 @@
+#GeoEntity API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [All Countries Request](#all-countries-request)
+- [All Administrative Area Ones Request](#all-administrative-area-ones-request)
+- [All Administrative Area Twos Request](#all-administrative-area-twos-request)
+- [All Localities Request](#all-localities-request)
+- [All Postal Codes Request](#all-postal-codes-request)
+- [All Metropolitan Statistical Areas Request](#all-metropolitan-statistical-areas-request)
+- [Versioning](#versioning)
+
+&nbsp;
+----------------------------
+#Description
+
+This service provides aggregated geography entities. Currently the following requests could be made.
+- Request all countries in the world.
+- Request all administrative area ones in a specific country.
```

Comment:
This one should be under your header on the next line, since it's country-specific. Also, I would rephrase these as "first-level administrative areas" etc instead of "administrative area ones"/"twos"